styles/globals.css

```diff
@@ -70,7 +70,15 @@ html {
     }
   }
 }
+
 .footerBackground {
   background:#26374a url(../public/landscape.png) no-repeat right bottom;
 }
 
+.circlePosition {
+  position: Relative ;
+left: -12%;
```

Comment:
Might want to get this indented ;)

Reply:
I missed this code I was planning to remove it. Its not used. ill fix it now. Ty